packages/net/http/http_notify.pony

```diff
@@ -45,6 +45,11 @@ interface tag HTTPSession
     submission of the HTTP message is complete.
     """
 
+  be dispose()
```

Comment:
It is not a notify method.  The file `http_notify` defines all the public "interfaces" to the HTTP package.  One is a real "notifier" interface, one is for a class factory (both of these the user has to implement) and the last is the generic interface to `HTTPSession`.  The new `dispose` method is to the `HTTPSession` actors, which is implemented twice in the package - once for clients and once for servers.  (The have identical interfaces but do different things, as the HTTP protocol is symmetrical in operation)

Perhaps what we need here is a different name for this file.